src/main/scala/jp/ne/opt/redshiftfake/parse/BaseParser.scala

```diff
@@ -7,6 +7,9 @@ import scala.util.parsing.combinator.RegexParsers
 
 trait BaseParser extends RegexParsers {
   val identifier = """[_a-zA-Z]\w*"""
+  val quotedIdentifier = """\"?[_a-zA-Z]\w*\"?""".r ^^ {
```

Comment:
[IMO] It would be better to reuse `identifier`.
`val quotedIdentifier = """\"?$identifier\"?""".r`